src/main/java/com/commercetools/project/sync/util/SyncUtils.java

```diff
@@ -53,13 +53,15 @@ public static String getApplicationVersion() {
       updateActionsString =
           updateActions.stream().map(Object::toString).collect(Collectors.joining(","));
     }
-    logger.error(
-        format(
-            "Error when trying to sync %s. Existing key: %s. Update actions: %s",
-            resourceName,
-            resource.map(WithKey::getKey).orElse(IDENTIFIER_NOT_PRESENT),
-            updateActionsString),
-        exception);
+    if (logger.isErrorEnabled()) {
```

Comment:
What if Error is not enabled? Do you know when can this happen?

Reply:
Error severity is enabled by default, 

Here I added it because of PMD validations, this is called guarded logs, when I upgraded the version of PMD it's changed so instead of reducing check levels I added this checks.

If not added PMD will report the warning below:
> GuardLogStatement:	Logger calls should be surrounded by log level guards.